task2_Gal_Malka/src/Position.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Position.h"
+
+Position::Position(const string& position){ 
+	row = int(position[0] - 'a');
```

Comment:
You should check the value of position, so your code will be aware of edge cases and deal with them. Also, I would expect some boundary check to the values of `row` and `col` in `Position`